### PR TITLE
New style of About View Controller.

### PR DESCRIPTION
**Pull request by ArkadiuszBanas**
I've decided to divide this task into two parts:
- applying new style to current controller organization. 
- Adding new FB/TW buttons and description on top. 
This is first part of task. 

Note: I've change separators height from 15px to 16px. Because operating on CGSizes like 7.5px could mess something up. 

Scrennshot: https://www.dropbox.com/s/2jykfedgoo75k2x/Screenshot%202016-02-15%2014.06.03.png?dl=0

### CODE REVIEW

File: BuyPolish/Pola/UI/ProductSearch/About/BPAboutViewControllerCell.m [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+CGFloat const WHITE_BACKGROUND_VERTICAL_MARGIN = 16.0f;
+CGFloat const LABEL_X_OFFSET = 20.0f;
+
+@interface BPAboutViewControllerCell ()
+@property (nonatomic) UIView *whiteBackgroundView;
+@end
+
+@implementation BPAboutViewControllerCell
+
+- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
```
Comment (paweljaneczek) [inline BuyPolish/Pola/UI/ProductSearch/About/BPAboutViewControllerCell.m]: There should be no option of creating cell with different style than `UITableViewCellStyleDefault`, so trim constructor to `initWithReuseIdentifier:` and make `[super initWithStyle:UITableViewCellStyleDefault reuseIdentifier:reuseIdentifier]` 2 lines below

**Comment by paweljaneczek** [comment]
Looks really clean and nice :+1: Only one small comment

**Comment by ArkadiuszBanas** [comment]
Hi Paweł, 
Done :)

**Comment by paweljaneczek** [comment]
:+1:

**Pull request merged**